src/components/editor/property/Literal.jsx

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import React from 'react'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import PropTypes from 'prop-types'
+import { removeItem } from 'actions/index'
+import LanguageButton from './LanguageButton'
+import { findNode } from 'selectors/resourceSelectors'
+
+const Literal = (props) => {
+  const handleEditClick = () => {
+    props.handleEdit(props.item.content)
+    props.removeItem(props.reduxPath)
+  }
+
+  const itemId = props.item.id
+
+  return (<div id="userInput">
```

Comment:
ids here and below should be unique.

Reply:
This ID was here before and the existing tests depend on it